gcloud/bigquery/table.py

```diff
@@ -761,8 +762,9 @@ def _bool_from_json(value, field):
 
 def _datetime_from_json(value, field):
     if _not_null(value, field):
-        # Field value will be in milliseconds.
-        return _datetime_from_microseconds(1000.0 * float(value))
+        # value will be a float in seconds, to microsecond precision, in UTC.
+        stamp = _datetime_from_microseconds(1e6 * float(value))
+        return stamp.replace(tzinfo=UTC)
```

Comment:
@tseaver `UTC` is [included][1] in `_datetime_from_microseconds` since it uses the [epoch][2]

[1]: https://github.com/GoogleCloudPlatform/gcloud-python/blob/61abd0c8ce00db9784d76589ba7e57c315e32b83/gcloud/_helpers.py#L234
[2]: https://github.com/GoogleCloudPlatform/gcloud-python/blob/61abd0c8ce00db9784d76589ba7e57c315e32b83/gcloud/_helpers.py#L273